uploader.json

```diff
@@ -2539,6 +2539,16 @@
             ],
             "symbolic_name": "ssh_foreman_proxy_config"
         },
+        {
+            "file": "/var/log/containers/swift/swift.log",
+            "pattern": [],
+            "symbolic_name": "swift_log"
+        },
+        {
+            "file": "var/log/swift/swift.log",
```

Comment:
@akshay196 your missing a leading `/` here.

Reply:
@bfahr Fixed. Thanks for pointing.